dapp/src/utils/connectors.js

```diff
@@ -52,5 +61,37 @@ export const connectorNameIconMap = {
   WalletConnect: 'walletconnect-icon.svg',
 }
 
+export const connectorsByName = {
```

Comment:
This is probably a result of a bad merge?

Reply:
What should be removed/edited here?

Reply:
The `connectorsByName` has just been refactored to `connectorNameIconMap` while you are working on this PR: 
https://github.com/OriginProtocol/origin-dollar/blob/master/dapp/src/utils/connectors.js#L48-L53